### ISSUE TITLE
Refactor host listeners from meComponent to keyboard\map component

### ISSUE THREAD

**Issue by katzi0** [Webiks/GeoStrike#196]

**Comment by katzi0** [comment]
host listeners in meComponent causing side affects regrading flight mode.
should cancel host listeners and find another way to detect changes in flight or integrate those listeners in keyboard component.

Et: 5 hours

**Comment by OmKoren** [comment]
@katzi0 why is the priority high?
Are we sure that this is what cause the performance issues?

**Comment by katzi0** [comment]
@OmKoren no. the main problem is the side effects caused by the combination of the host-listeners and the interpolation function.
